code/obj/item/kitchen.dm

```diff
@@ -680,6 +680,9 @@ TRAYS
 		if(ordered_contents.len == max_food)
 			boutput(user, "That won't fit, \the [src] is too full!")
 			return
+		if(istype(W, /obj/item/reagent_containers/food/snacks/prison_loaf))
```

Comment:
Should this just gate on `W.w_class`, or are there things we want this to work for?

Reply:
I've thought about it for a second but personally I don't really mind if a chef loads an enormous cake on a plate (do they even have a larger w.class?) and transports it that way. The loaves are the issue, because in case of higher tier they're very abusable

Reply:
yea, that actually makes more sense now that I think of it